cookbooks/kinematically_driven_subduction_2d/composition_trms_statistics.h

```diff
@@ -0,0 +1,75 @@
+/*
+  Copyright (C) 2011 - 2023 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__postprocess_composition_trms_statistics_h
+#define __aspect__postprocess_composition_trms_statistics_h
+
```

Comment:
Double underscores are reserved for compiler purposes. We switched a while ago, but this branch was created before that time.
```suggestion
#ifndef _aspect_postprocess_composition_trms_statistics_h
#define _aspect_postprocess_composition_trms_statistics_h

```